ote/src/clj/ote/services/routes.clj

```diff
@@ -137,18 +148,44 @@
            (delete! db ::transit/route {::transit/id id})
            id))))
 
+(defn link-interface
+  "This is a helper function for users. Links created route to given service.
+  Users can add interface by hand but this is much more convenient for them."
+  [db user  {is-linked? :is-linked?
+             service-id :service-id
+             operator-id :operator-id :as form-data}]
+  (let [interface-url (interface-url operator-id)]
+    (authorization/with-transport-operator-check
+      db user operator-id
+      #(do
```

Comment:
```suggestion
      #(if
```

Reply:
I'm leaving do because in if there is some action and the function return value is after the if.